src/RefractDataStructure.cc

```diff
@@ -719,6 +719,10 @@ namespace drafter
 
     refract::IElement* DataStructureToRefract(const snowcrash::DataStructure& dataStructure)
     {
+        if (dataStructure.empty()) {
+            return NULL;
+        }
+
```

Comment:
@klokane The issue which we need to fix here is that we do not have an [API namespace DataStructure](https://github.com/refractproject/refract-spec/blob/968cf6042757a365115ce88a5ddab6912e0e4a1e/namespaces/api-description-namespace.md#data-structure-element) here. Our AST directly embeds the MSON Refract.